astropy/utils/xml/iterparser.py

```diff
@@ -69,7 +71,7 @@ def _convert_to_fd_or_read_function(fd):
                 yield new_fd.read
 
 
-def _fast_iterparse(fd, buffersize=2**10):
+def _slow_iterparse(fd, buffersize=2**10):
```

Comment:
Thanks for not deleting this function altogether. I think this is still good to keep for reference or debugging.

Reply:
It's still being used: https://github.com/astropy/astropy/blob/6219b8bcf05cddc1e878f3058c396a43420064c6/astropy/utils/xml/iterparser.py#L158